infrastructure/io/src/main/java/tech/pegasys/teku/infrastructure/io/SyncDataAccessor.java

```diff
@@ -52,9 +52,11 @@
    * @exception IOException if an IO error occurs while writing
    */
   public void syncedWrite(final Path path, final Bytes data) throws IOException {
-    final File parentDirectory = path.getParent().toFile();
-    if (!parentDirectory.mkdirs() && !parentDirectory.isDirectory()) {
-      throw new IOException("Unable to create directory " + parentDirectory);
+    if (path.getParent() != null) {
+      final File parentDirectory = path.getParent().toFile();
+      if (!parentDirectory.mkdirs() && !parentDirectory.isDirectory()) {
+        throw new IOException("Unable to create directory " + parentDirectory);
+      }
```

Comment:
I'd suggest using `path.toAbsolutePath()` to get an absolute path before attempting to get the parent.  It still may have no parent if it's the root path but if it's a relative path the absolute version may have a parent when the relative doesn't.